dev/test_models.py

```diff
@@ -138,7 +137,6 @@ class TestError(unittest.TestCase):
     @parameterized.expand(parse_tests(load_test_data("error")), skip_on_empty=True)
     @unittest.skipIf(not _RUN_ERROR_TESTS, "Skipping error tests")
     def test(self, key, out_expected):
-        env.setup_env()
```

Comment:
As we discussed, this will likely break running the tests on fb stack. We should be able to fix by making a fb dir here and overriding the test to first register manifold handler